applications/protocols_serialization/client/sample.yaml

```diff
@@ -35,3 +35,14 @@ tests:
     integration_platforms:
       - nrf52840dk/nrf52840
       - nrf54l15pdk/nrf54l15/cpuapp
+  applications.protocols_serialization.client.all:
```

Comment:
Should we just extend the previous `.rpc` test to reduce the number of builds?

Reply:
I'm not sure about this, we can talk about how to tackle this on a daily.

Reply:
@tomaszkob89 any thoughts?

Reply:
Feel free to disregard this for now. I just thought it would be good to only build combinations used by our CI (others might be built in nightly only). But this optimization can wait.